.github/workflows/release.yml

```diff
@@ -12,25 +12,16 @@ jobs:
     if: ${{ startsWith(github.ref, 'refs/tags/v') || startsWith(github.ref, 'refs/heads/release/v') }}
     runs-on: ubuntu-20.04
     steps:
-      - name: '❄ Wait for Hydra build'
-        uses: rvl/hydra-build-products-action@master
-        id: hydra
-        with:
-          hydra: 'https://hydra.iohk.io'
-          jobs: 'linux.musl.cardano-wallet-linux64 macos.intel.cardano-wallet-macos-intel linux.windows.cardano-wallet-win64'
-
-      - name: '🍒 Fetch release files'
-        run: |
-          wget ${{ steps.hydra.outputs.buildProducts }}
-
+      # Creates an empty release draft
+      # TODO: get artifacts from Buildkite to be attached to the draft
       - name: '🚀 Release'
         uses: softprops/action-gh-release@v1
         with:
           draft: true
-          fail_on_unmatched_files: true
-          files: |
-            *.tar.gz
-            *.zip
+          # fail_on_unmatched_files: true
+          # files: |
+          #   *.tar.gz
+          #   *.zip
```

Comment:
What's the idea of disabling this block? Please leave a comment that explains:
- Why is it currently commented out;
- What is needed to uncomment it;
- Link to a JIRA task that tracks the work needed to be done;

Reply:
added jira ticket

Reply:
Please reference the JIRA ticket you've created in the comment before the block, this would help to correlate them later. Thanks!

Reply:
Yes, it's there